Sources/SwiftParser/Types.swift

```diff
@@ -639,14 +643,88 @@ extension Parser {
 
 extension Parser.Lookahead {
   mutating func canParseType() -> Bool {
+    guard self.canParseTypeScalar() else {
+      return false
+    }
+
+    if self.currentToken.isEllipsis {
+      self.consumeAnyToken()
+    }
+
+    return true
+  }
+
+  mutating func skipTypeAttributeList() {
+    var specifierProgress = LoopProgressCondition()
+    // TODO: Can we model isolated/_const so that they're specified in both canParse* and parse*?
```

Comment:
I hope to be working on exactly that in the next couple of days. 🤞

Reply:
Note that @bnbarham authored this TODO; I just hoisted the method closer to its new client.